### DISCUSSION TITLE
VS Code key bindings?

### DISCUSSION THREAD

**Discussion by fedorinoGore** [NvChad/NvChad#1757] [Q&A]
I'm trying to create an alternative to my VS Code setup that I could use to debug some of the code remotely.

Just found out that learning the Vim keymaps may take much longer than expected and that after working with Vim I can't seamlessly switch back to VS Code as my "finger" memory is trying to mix Vim and VS Code key bindings.

Please advise - is there any way to replace native Vim key bindings with VS Code bindings?
Are there any technical issues that prevent it besides snobbery?

**Comment by siduck** [comment]
@fedorinoGore  what bindings are you talking about? Idk about vscode bindings :thinking:

**Reply by fedorinoGore** [reply]
Yes. I'm actually want to use same keyboard shortcuts that I'm using for VS Code with Neovim and NvChad config. 
Is it possible?

**Reply by siduck** [reply]
@fedorinoGore read the mapping section in our docs, you have to make new mappings.
